apps/src/code-studio/levels/postOnLoad.js

```diff
@@ -18,11 +18,18 @@ export function postMilestoneForPageLoad() {
   // this will just be { response: 'ok', result: true }
   const result = getResult();
 
+  // callback here really means "url that we post to"
+  const postUrl = appOptions.dialog.callback;
+  if (!postUrl) {
+    // Don't bother trying to post if we don't have a url to post to. One known
+    // case where this happens is if I go to a /levels/<levelnum> page
+    return;
+  }
```

Comment:
I got annoyed seeing JS errors when we tried/failed to post on load when going to a levels/levelnum page, so I made it not do that.